packages/react-aria-components/example/index.css

```diff
@@ -175,3 +175,17 @@ html {
     }
   }
 }
+
+.dropzone {
+  border: 2px solid white;
+  margin: 20px;
+  padding: 20px;
+
+  &[data-focus-visible][data-focused] {
+    border-color: blue;
+  }
+
+  &[data-hovered]{
```

Comment:
could we do `data-hovered={isHovered || isDropTarget || undefined} `?

Reply:
I don't think we want to always show the "this is a drop target that you're dropping on" when the user is just hovering without dragging. I feel like that would be confusing because it'd lead them to think that they were dragging something.

Reply:
Maybe something like:

```tsx
data-drop-target={isDropTarget || undefined}
```